sanic/worker/container.py

```diff
@@ -0,0 +1,17 @@
+from sanic.worker.loader import AppLoader
+
+
+class AppContainer:
+    def __init__(self, loader: AppLoader) -> None:
+        self.loader = loader
+
+    def prepare(self, *apps) -> None:
+        for app in apps:
+            app.prepare(**app._early_prepare)
+
+    def serve(self) -> None:
+        from sanic import Sanic
+
+        primary = self.loader.load()
+        self.prepare(primary)
```

Comment:
Oops, this file was the beginning of a different idea I had. Doesn't belong in this PR. I'll remove.